public/css/user.less

```diff
@@ -0,0 +1,5 @@
+#aposLoginHcaptcha {
+  width: 164px;
+  margin: auto;
+  margin-bottom: 2em;
+}
```

Comment:
wow, it's been so long I haven't seen ids in CSS, I thought at first that `#` was a less specific character 😅